### PR TITLE
Frontend lobby selector prototype

### PR DESCRIPTION
**Pull request by Quickzand**

### CODE REVIEW

**Review by maximusmith529 — CHANGES_REQUESTED** [review]
Please make all comments follow the same formatting and capitalization practices

File: POOSDKer.xcodeproj/xcshareddata/xcschemes/Network Testing.xcscheme [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      buildImplicitDependencies = "YES">
+      <BuildActionEntries>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "YES"
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
```
Comment (aintzanepm) [inline POOSDKer.xcodeproj/xcshareddata/xcschemes/Network Testing.xcscheme:16]: Is there any reason why this and the "internal" on line 40 are lowercase, while everything else is proper case?

File: POOSDKer/Backend/NetworkingController.swift [hunk 2]
```diff
@@ -6,105 +6,263 @@
...
-
-            listener?.start(queue: .main)
+        }
+        
+        // You can decide whether to accept the invitation based on the received information
+        invitationHandler(true, mcSession)
+    }
+    
+    
+    // !! It may make sense in the future to abstract out the process of broadcasting
```
Comment (aintzanepm) [inline POOSDKer/Backend/NetworkingController.swift:139]: Do the exclamation marks have a specific meaning? Maybe we should settle on a specific "TO-DO", "URGENT", "FUT. TASK", etc. standard to mark sections.

File: POOSDKer.xcodeproj/xcshareddata/xcschemes/Network Testing.xcscheme (hunk 1 above)
Comment (Quickzand) [reply POOSDKer.xcodeproj/xcshareddata/xcschemes/Network Testing.xcscheme:16]: no clue! Its auto generated by xcode

**Review by aintzanepm — CHANGES_REQUESTED** [review]
I second with Max's "comment standard" comment, note my reviews and besides that LGTM!

File: POOSDKer/Backend/NetworkingController.swift (hunk 2 above)
Comment (Quickzand) [reply POOSDKer/Backend/NetworkingController.swift:139]: Xcode has different ways to markup code with comments. !! is one. There is also MARK: that created a mark

Comment (aintzanepm) [reply POOSDKer/Backend/NetworkingController.swift:139]: Is there any documentation you can link for this? Would be helpful

Comment (Quickzand) [reply POOSDKer/Backend/NetworkingController.swift:139]: https://stackoverflow.com/questions/7600435/what-source-comments-does-xcode-recognize-as-tags


There are quite a few not documented but this thread has a lot of em. Ofc theres no offical docs 🙃

**Review by aintzanepm — APPROVED** [review]
LGTM!

**Review by maximusmith529 — APPROVED** [review]
Lgtm

**Pull request merged**